documentation/src/main/asciidoc/reference/integrations.asciidoc

```diff
@@ -0,0 +1,56 @@
+[[integrations]]
+= Standards and integrations
+
+[[other-integrations-jakarta]]
+== Jakarta EE
+
+include::components/incubating-warning.asciidoc[]
+
+Hibernate Search includes experimental support for link:{jakartaUrl}[Jakarta EE],
+and it requires only one small change:
+when declaring the dependencies of your project,
+you must add `-jakarta` to some artifact identifiers.
+
+For example, an application using Hibernate ORM and the Elasticsearch backend
+will need to update its dependencies as follows:
+
+[source, XML, subs="+attributes"]
+----
+<dependency>
+   <groupId>org.hibernate</groupId>
+   <artifactId>hibernate-core-jakarta</artifactId> <!--1-->
+   <version>{hibernateVersion}</version>
+</dependency>
+<dependency>
+   <groupId>org.hibernate.search</groupId>
+   <artifactId></artifactId>
+   <artifactId>hibernate-search-mapper-orm-jakarta</artifactId> <!--2-->
+   <version>{hibernateSearchVersion}</version>
+</dependency>
+<dependency>
+   <groupId>org.hibernate.search</groupId>
+   <artifactId>hibernate-search-backend-elasticsearch</artifactId> <!--3-->
+   <version>{hibernateSearchVersion}</version>
+</dependency>
+----
+<1> Replaces `hibernate-core`.
+<2> Replaces `hibernate-search-mapper-orm`.
+<3> No replacement necessary: this artifact does not rely on Java EE.
+
+All artifacts relying directly or indirectly on Java EE must be replaced with their Jakarta counterpart.
+This includes in particular:
+
+* https://in.relation.to/2021/06/04/hibernate-is-jakarta-jpa-2/#get-it[Hibernate ORM artifacts]
+* `hibernate-search-mapper-orm` => `hibernate-search-mapper-orm-jakarta`
+* `hibernate-search-mapper-orm-batch-jsr352-core` => `hibernate-search-mapper-orm-batch-jsr352-core-jakarta`
+* ...
```

Comment:
I think that it is better to list all the artifacts that must be replaced with their Jakarta counterpart.

Reply:
True, but we will likely forget to add them here when we add more artifacts, so I'd rather be clear from the start that the list may be incomplete.